website/static/css/custom.css

```diff
@@ -351,6 +379,25 @@ span.arrow::after {
     color: #0072CE;
 }
 
+.navGroupCategoryTitle {
+    font-family: BwSeidoRound-Medium,sans-serif;
```

Comment:
It seems that most 'font-family' style is to add various levels of font weight. We could add font families at a higher level (body, etc.) and add a specific 'font-weight' to elements. We can also utilize SASS mixins and include all font-related style and make it easier to remember complicated font-family names, e.g '@include roundMedium'. Just worth thinking about.

Reply:
punted on this pending discussion with you when we have time

Reply:
📅 scheduled for later